en/micro-integrator/docs/references/connectors/gmail-connector/configuring-gmail-api.md

```diff
@@ -0,0 +1,52 @@
+## Creating the Client ID and Client Secret
+
+1. Navigate to [API Credentials Page](https://console.developers.google.com/projectselector/apis/credentials) and sign in with your Google account.
+
+2. Click on **Select a Project** and click **NEW PROJECT**, to create a project.
+ <img src="/assets/img/connectors/create-project.png" title="Creating a new Project" width="800" alt="Creating a new Project" />
```

Comment:
There are too many screenshots to showcase simple things like navigation. Let's only include screenshots that are absolutely necessary.

Reply:
Removed the screenshot in step 3.

Reply:
Do you need the one in step 5?

Reply:
Since it's a new interface, I kept that.